include/CppUTest/CppUTestConfig.h

```diff
@@ -200,11 +200,13 @@
 #endif
 
 /* Support for "long long" type */
+#ifdef CPPUTEST_HAVE_LONG_LONG_INT
+#define CPPUTEST_USE_LONG_LONG 1
+#endif
```

Comment:
This is exactly the sort of auto-detection that was objected to in my first revision of the PR...